schema/observability/traces/traces-1.0.0.mapping

```diff
@@ -105,26 +105,6 @@
             }
           }
         },
-        "attributes": {
-          "type": "object",
-          "data_stream": {
-            "properties": {
-                "dataset": {
-                  "ignore_above": 128,
-                  "type": "keyword"
-                },
-                "namespace": {
-                  "ignore_above": 128,
-                  "type": "keyword"
-                },
-                "type": {
-                  "ignore_above": 56,
-                  "type": "keyword"
-                }
-              }
-            }
-          }
-        },
```

Comment:
Will this break the OpenSearch OTEL exporter: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/5698c9def9352a1ea5ad9bc307265a4a04de9c1f/exporter/opensearchexporter/encoder.go#L187-L200

Reply:
Yes - I'll add this change under a new minor version